docs/spec_draft.md

```diff
@@ -2279,6 +2280,85 @@ where `i[d] = j[permutation[d]]`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.triangular_solve
+
+### Semantics
+
+Solves systems of linear equations of a batch of matrices with lower or upper
+triangular coefficients by forward- or back-substitution.
```

Comment:
Proposed rewording:

```
Solves batches of systems of linear equations with lower or upper triangular
coefficient matrices.
```

Feedback:
  1. "by forward- or back-substitution". Is this an essential part of semantics? If not, let's drop this and let implementations decide how they want to solve these systems of linear equations.
  2. I don't think it's "systems of equations of a batch of ...". I think it's "batches of systems of equations of ...".
  3. "coefficient matrix" is [a term of art](https://en.wikipedia.org/wiki/Coefficient_matrix), so let's keep it unchanged.